pom.xml

```diff
@@ -718,7 +724,7 @@
             <plugin>
                 <!-- https://stackoverflow.com/questions/46177921/how-to-run-unit-tests-in-excludedgroups-in-maven -->
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.0</version>
+                <version>3.0.0-M1</version>
```

Comment:
I can't find good resources about what might be affected by this version change, as this is really new. The official release notes just show a few improvements/fixes, so everything should be just fine.

Dunno what IQSS policy is on this. You could upgrade to 2.22.1, too.

Reply:
The reason for the upgrade is that the maven build was not passing on my pc, after reaserch I found out that it was realted to OpenJDK 8 bug for Debian and at the same time some bug in the surefire plugin which was exactly fixed in this version.